wof/examples/flask/odm2/timeseries/odm2_timeseries_dao.py

```diff
@@ -1,57 +1,52 @@
 from __future__ import (absolute_import, division, print_function)
 
-from datetime import datetime
+from dateutil.parser import parse
+
 from sqlalchemy import create_engine
 from sqlalchemy.orm import scoped_session, sessionmaker
-from sqlalchemy.sql import and_
-
-from dateutil.parser import parse
 
 from wof.dao import BaseDao
 import wof.examples.flask.odm2.timeseries.sqlalch_odm2_models as model
-from odm2api.ODM2 import models as odm2_models
-from sqlalchemy.orm import aliased
-from sqlalchemy import or_
-from sqlalchemy import and_
-from sqlalchemy.sql import func
 
-from sqlalchemy.orm import with_polymorphic
+from odm2api.ODM2 import models as odm2_models
 
-import re
 
 class Odm2Dao(BaseDao):
 
     def __init__(self, db_connection_string):
 
-        self.engine = create_engine(db_connection_string, convert_unicode=True,
-            #pool_size=100
-                                    )
-        odm2_models.setSchema(self.engine)
         # Default application pool size is 5. Use 100 to improve performance.
-        self.db_session = scoped_session(sessionmaker(
-            autocommit=False, autoflush=False, bind=self.engine))
+        self.engine = create_engine(
+            db_connection_string,
+            convert_unicode=True,
+            # pool_size=100
+        )
 
+        odm2_models.setSchema(self.engine)
 
+        Session = scoped_session(
+            sessionmaker(
+                autocommit=False,
+                autoflush=False,
+                bind=self.engine
+            )
+        )
 
-        #odm2_models.Base.query = self.db_session.query_property()
+        self.db_session = Session()
 
     def __del__(self):
+
         self.db_session.close()
 
     def get_all_sites(self):
-        s_rArr = self.db_session.query(odm2_models.Sites,odm2_models.TimeSeriesResults).\
+
+        s_rArr = self.db_session.query(odm2_models.Sites,
+                                       odm2_models.TimeSeriesResults).\
```

Comment:
Ignore what I said. Not worth discussing or changing right now. 

But just FYI, this is what we tweaked and experimented with over a week ago in the notebook you lost.